packages/search-in-workspace/src/common/search-in-workspace-service.ts

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (C) 2017 Erisson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable, inject } from "inversify";
+import { SearchInWorkspaceServer, SearchInWorkspaceClient, SearchInWorkspaceResult, SearchInWorkspaceOptions } from "../common/search-in-workspace-interface";
+import { WorkspaceService } from "@theia/workspace/lib/browser";
+import URI from "@theia/core/lib/common/uri";
+import { ILogger } from "@theia/core";
+
+/* Class that will receive the search results from the server.  This is separate
+   from the SearchInWorkspaceService class only to avoid a cycle in the
+   dependency injection.  */
```

Comment:
Such API comments should be formatted like so:
```
/** 
 * Class that will receive the search results from the server.  This is separate
 * from the SearchInWorkspaceService class only to avoid a cycle in the
 * dependency injection.  
 */
```

Reply:
Ok.